### PR TITLE
[Do not merge] CDN in all languages in conf

### PR DESCRIPTION
**Pull request by lmazuel**
This PR to show what should look like the common process when all languages will be plugged. This is for CSharp, Java, Python, Node, Go and Ruby

- [x] @shahabhijeet I moved the CSharp configuration into `readme.csharp.md`. This is absolutely transparent, Autorest manages these sub-files automatically.
- [ ] @daschult I moved the Java configuration into `readme.java.md`. This is absolutely transparent, Autorest manages these sub-files automatically.
- [x] @marstr I moved the Go configuration into `readme.go.md`. This is absolutely transparent, Autorest manages these sub-files automatically.
- [x] @amarzavery I manually wrote `readme.nodejs.md` from the info you gave to me. Please review. :)
- [ ] @sarangan12 I manually wrote `readme.ruby.md` from your current swaggertosdk conf into the new format. Please review this is equivalent. This is not ready, since the new system cannot handle correctly `generated_relative_base_directory` that you need. I have to think about this a little more.

Will keep this PR open until all languages gives good result for the automation system. Then, CDN will be an example on how to write the new generation format.

Edit: adding checkbox for "tested and correct".

### CODE REVIEW

**Comment by lmazuel** [comment]
@AutorestCI rebuild azure-sdk-for-go

File: specification/cdn/resource-manager/readme.nodejs.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+These settings apply only when `--nodejs` is specified on the command line.
+Please also specify `--node-sdks-folder=<path to the root directory of your azure-sdk-for-node clone>`.
+
+``` yaml $(nodejs)
+nodejs:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
+  package-name: azure-mgmt-cdn
```
Comment (amarzavery) [inline specification/cdn/resource-manager/readme.nodejs.md]: azure-arm-cdn

**Review by amarzavery — APPROVED** [review]

**Review by amarzavery — APPROVED** [review]

**Comment by lmazuel** [comment]
@AutorestCI rebuild azure-sdk-for-go

**Comment by lmazuel** [comment]
@marstr See the latest message (i.e. bash/sh fixed). We're making progress :)

**Comment by lmazuel** [comment]
Merging, I don't have feedback from Java but everything else should be kept anyway.

**Pull request merged**